h2o-genmodel/src/main/java/hex/genmodel/ModelMojoReader.java

```diff
@@ -49,6 +49,13 @@ public static MojoModel readFrom(MojoReaderBackend reader) throws IOException {
 
   protected abstract M makeModel(String[] columns, String[][] domains, String responseColumn);
 
+  /**
+   * Version of the mojo file produced. Follows the <code>major.minor</code>
```

Comment:
Is this version of the mojo file?

Reply:
I think this version is associated with genmodel.jar and represented the h2o-3 version that actually generated the genmodel.jar.

Reply:
yeah, but the documentation of this method is not correct in my opinion - it is not a version of "a mojo file produced" because this class doesn't produce any mojo file ;) also it is not the version of the mojo file that is being parsed by the reader; that is why I think we need to revise the method documentation

Reply:
perhaps even rename, this comparison `_model._mojo_version > Double.parseDouble(mojoVersion())` is hard to understand

would be easier if it was maxSupportedMojoVersion() instead or something like that - then there is no question about what the statement is doing. wdyt?

Reply:
Oh, that is my mistake. I only copied the doc from the same method from `MojoWriter,` and I didn't check the text. I will change it.

And I also agree with rename the check method. Thanks for catching it.